README.md

```diff
@@ -23,10 +23,21 @@ specification to describe this ability as well, and move the document to the sta
   * todo
 
 ## Goals
-  * todo
+* Enable a web app to register tasks to be run periodically, with a retry logic baked in, which has an exponential backoff.
+* Provide the web app a way to query the list of previously registered tasks.
+* Enable the web app to unregister previously registered tasks.
+* Supported platforms: Android, Windows, Linux, ChromeOS, Mac. 
 
 ## Non-goals
-  * todo
+* Triggering events at a specific time is an explicit non-goal. The alarms API enables that.
+* Notifying the user for each periodic Sync event. The user can be presented with updated content
+  when they next visit the website.
```

Comment:
This is a non-goal of our implementation, the standard doesn't care.

Reply:
Ack, removed.